tests/test_cse.py

```diff
@@ -0,0 +1,51 @@
+import torch
+import torch_ttnn
+import ttnn
+import unittest
+from torch.fx.passes.dialect.common.cse_pass import CSEPass
+
+
+class AddModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, x):
+        return x + x
+
+    def input_shapes(self):
+        return [(4, 4)]
+
+
+class TestModules(unittest.TestCase):
+    def setUp(self):
+        # Open device 0
+        self.device: ttnn.Device = ttnn.open_device(device_id=0)
+
+    def tearDown(self):
+        # Close the device
+        ttnn.close_device(self.device)
+
+    def test_add(self):
+        m = AddModule()
+        input_shapes = m.input_shapes()
+        inputs = [torch.rand(shape, dtype=torch.bfloat16) for shape in input_shapes]
+        result_before = m.forward(*inputs)
+        option = torch_ttnn.TenstorrentBackendOption(device=self.device)
+        # The compilation is lazy, so we need to run forward once to trigger the compilation
+        m = torch.compile(m, backend="ttnn", options=option)
+        result_after = m.forward(*inputs)
+        self.assertEqual(1, len(option._out_fx_graphs))
+        option._out_fx_graphs[0].print_tabular()
+        # Check the graph has be rewritten and contain ttnn ops
+        nodes = list(option._out_fx_graphs[0].nodes)
+        self.assertEqual(nodes[1 + 0].target, ttnn.from_torch)
+        self.assertEqual(nodes[1 + 1].target, ttnn.to_layout)
+        self.assertEqual(nodes[1 + 2].target, ttnn.to_device)
+        self.assertEqual(nodes[1 + 3].target, ttnn.to_layout)
+        self.assertEqual(nodes[1 + 4].target, ttnn.to_device)
+        self.assertEqual(nodes[6].target, ttnn.add)
+        self.assertEqual(nodes[7 + 0].target, ttnn.from_device)
+        self.assertEqual(nodes[7 + 1].target, ttnn.to_layout)
+        self.assertEqual(nodes[7 + 2].target, ttnn.to_torch)
```

Comment:
@kevinwuTT, these from_device, to_layout, to_torch parts have to be removed, right?

Reply:
Yeah, currently the tests are no longer checking for these ops except for maybe some special cases.

Reply:
@jdh8 @yocosky , please remove redundant checks. 

See existing tests for an example
https://github.com/tenstorrent/pytorch2.0_ttnn/tree/main/tests/lowering

Let me know if you have any concerns.

Our main motivation is next. 
* We don't need to separately call from_torch, to_layout, to_device. This can be a single call. Same with from_device, to_layout, to_torch. Ops like from_torch and to_torch can handle most cases themself.
* Existence of those movement ops does not really check what the lowering of a particular operation does. And we want the test to focus on that
* Having those movement op checks unnecessary complicates maintenance

Reply:
Done!